assets/js/pages/HostsList/HostsList.test.jsx

```diff
@@ -121,6 +124,30 @@ describe('HostsLists component', () => {
         ).toBeVisible()
       );
     });
+
+    it('should show only unique SIDs', async () => {
+      const host = hostFactory.build();
+      const duplicateSID = faker.string.alpha({ casing: 'upper', count: 3 });
+      const sapInstances = sapSystemApplicationInstanceFactory.buildList(2, {
+        sid: duplicateSID,
+        host_id: host.id,
+      });
+
+      const state = {
+        ...defaultInitialState,
+        hostsList: {
+          hosts: [host],
+        },
+        sapSystemsList: {
```

Comment:
let's add some database instances with the same SID as well, as we need to have the same logic for bot